.github/workflows/ci-ultra-tests.yml

```diff
@@ -1,6 +1,14 @@
 name: ULTRA CI Base Tests
 
-on: [push, pull_request]
+on: 
+  push:
+    branches:
+      - ultra-**
+      - ultra_**
```

Comment:
![image](https://user-images.githubusercontent.com/36685292/115137636-38949e80-9ff5-11eb-92f6-58b494063a87.png)
I tried to use ultra[-_]** or 'ultra[-_]**' glob matching but got this error, so I switched to the way above

Reply:
Oh strange, I had included a slash to match branches that begin with `ultra/` (in addition to `ultra-` and `ultra_`), and it seemed to work with the pattern `ultra[-_/]**`. If `ultra/` is an ULTRA branch name that we want to include, maybe give that pattern a try? Or include `ultra/` in the list of branches to include.

Reply:
yes I should also add a ultra / ** here. Though my main question is the pattern ultra[ - _ /]** might not have worked. I based off ultra-develop branch then pushed a branch named 'smarts-branch-test-ci', the smarts-ci-base-test were not running on this test branch. So I think the glob expression might not been working
![image](https://user-images.githubusercontent.com/36685292/115249625-cad09b80-a0f6-11eb-9ab8-8418c4557c6e.png)
![image](https://user-images.githubusercontent.com/36685292/115249684-dae87b00-a0f6-11eb-9e71-e4c57be29bfb.png)

Reply:
![image](https://user-images.githubusercontent.com/36685292/115250910-061f9a00-a0f8-11eb-9cfc-849f269967a7.png)

Reply:
Oh okay thank you for letting me know. I will investigate this as well.

Reply:
> ![image](https://user-images.githubusercontent.com/36685292/115137636-38949e80-9ff5-11eb-92f6-58b494063a87.png)
> I tried to use ultra[ - _ ]** or 'ultra[ - _ ]**' glob matching but got this error, so I switched to the way above

If you are using glob wouldn't it be something along the lines of `[\-_]**` since `"-"` is a reserved character?

Reply:
ah makes sense

Reply:
![image](https://user-images.githubusercontent.com/36685292/115265466-3faad200-a105-11eb-8a8a-e0a9d164b749.png)
![image](https://user-images.githubusercontent.com/36685292/115265509-4b969400-a105-11eb-9bdc-1eac78ce43ff.png)
None of them worked actually. Will switch back to without using glob

Reply:
This looks like yaml is interacting badly with pattern matching.

Reply:
Okay good to know, I have opened #780 to fix the `ultra-develop` branch.